Source/Core/VideoCommon/VideoConfig.h

```diff
@@ -25,10 +25,12 @@
 
 enum AspectMode
 {
-	ASPECT_AUTO       = 0,
-	ASPECT_FORCE_16_9 = 1,
-	ASPECT_FORCE_4_3  = 2,
-	ASPECT_STRETCH    = 3,
+	ASPECT_AUTO        = 0,
+	ASPECT_FORCE_16_9  = 1,
+	ASPECT_FORCE_4_3   = 2,
+	ASPECT_STRETCH     = 3,
+	ASPECT_ANALOG      = 4,
+	ASPECT_ANALOG_WIDE = 5,
```

Comment:
I can do this.  I was trying to leave existing options in tact, but if it is okay to just get rid of them, then I will.